tests/scripts/run_tests.sh

```diff
@@ -82,6 +82,7 @@ run_frequent_api_pipeline_tests() {
     else
         if [[ $tt_arch == "wormhole_b0" ]]; then
             pytest  tests/tt_eager/python_api_testing/unit_testing/misc/test_all_gather.py -k nightly
+            pytest  tests/tt_eager/python_api_testing/unit_testing/misc/test_reduce_scatter.py -k nightly
```

Comment:
Is it possible you can factor out the nightly tests into separate files?

We want to reduce usage of `-k` per #7586

Reply:
Hey @tt-rkim, are you suggesting here that I make a `test_reduce_scatter_post_commit.py` and `test_reduce_scatter_nightly.py`? Are we planning on breaking off directories as well by test type or not necessarily?

Reply:
The original sources can continue being placed according to current patterns, but we're planning to put them as symlinks to designate pipelines more clearly.

Take a look at `tests/nightly`, they symlink to a bunch of stuff in other places